script/vsts/platforms/macos.yml

```diff
@@ -14,25 +14,22 @@ jobs:
     steps:
       - task: NodeTool@0
         inputs:
-          versionSpec: 12.13.1
-        displayName: Install Node.js 12.13.1
+          versionSpec: 12.4.0
+        displayName: Install Node.js 12.4.0
 
-      - script: npm install --global npm@6.12.1
-        displayName: Update npm
-
-      - task: 1ESLighthouseEng.PipelineArtifactCaching.RestoreCacheV1.RestoreCache@1
-        displayName: Restore node_modules cache
-        inputs:
-          keyfile: 'package.json, script/vsts/platforms/macos.yml, **/package-lock.json, !**/node_modules/**/package-lock.json, !**/.*/**/package-lock.json'
-          targetfolder: '**/node_modules, !**/node_modules/**/node_modules'
-          vstsFeed: 'bae1bc26-220d-43c7-a955-4de039370de2'
+      #- task: 1ESLighthouseEng.PipelineArtifactCaching.RestoreCacheV1.RestoreCache@1
+      #  displayName: Restore node_modules cache
+      #  inputs:
+      #    keyfile: 'package.json, script/vsts/platforms/macos.yml, **/package-lock.json, !**/node_modules/**/package-lock.json, !**/.*/**/package-lock.json'
+      #    targetfolder: '**/node_modules, !**/node_modules/**/node_modules'
+      #    vstsFeed: 'bae1bc26-220d-43c7-a955-4de039370de2'
```

Comment:
Probably want to un-comment this to re-enable the bootstrap cache on macOS CI.

_(I'm not sure if that would interrupt the current CI run if pushed right now, though!)_

Reply:
O thanks for the heads up! I will do that on my end, no worries